src/main/java/org/molgenis/calibratecadd/support/GavinUtils.java

```diff
@@ -18,13 +18,16 @@
 import java.util.zip.ZipFile;
 
 import static java.nio.charset.StandardCharsets.UTF_8;
+import static org.molgenis.data.annotation.entity.impl.gavin.GavinEntry.CADD_INDEX;
+import static org.molgenis.data.annotation.entity.impl.gavin.GavinEntry.PATHO_MAF_INDEX;
 
 public class GavinUtils
 {
 	private static final Logger LOG = LoggerFactory.getLogger(GavinUtils.class);
 
 	private static final String HEADER_PREFIX = "##";
 	private static final String PEDIGREE = "##PEDIGREE";
+	public static final String DEFAULT_KEY = "DEFAULT";
```

Comment:
Prefix with # or ## (VCF) to distinguish from a header line? How about `##DEFAULT_THRESHOLD=xxx`?